crates/client/rpc-core/src/lib.rs

```diff
@@ -254,6 +256,14 @@ pub enum RPCContractClass {
     ContractClass(SierraContractClass),
 }
 
+/// Starknet transaction for RPC serving.
+#[derive(Serialize, Deserialize, Clone, Debug, PartialEq)]
+pub struct RpcTransaction {
```

Comment:
I did a new struct to encapsulate internal data and be sure we can have loose coupling between the struct sent in the RPC response, and internal organization. Is that ok?